diff --git a/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js b/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js
index 4db9fee..2d4139b 100644
--- a/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js
+++ b/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js
@@ -50,6 +50,9 @@ function loadManifest($PATH, $$$ARGS) {
     return process.env.NEXT_BUILD_ID;
 	}
   ${returnManifests}
+  if ($PATH.endsWith("/server/subresource-integrity-manifest.json")) return {};
+  if ($PATH.endsWith("/server/prefetch-hints.json")) return {};
+  if (arguments[4]) return {};
   throw new Error(\`Unexpected loadManifest(\${$PATH}) call!\`);
 }`,
     };
